README.md

```diff
@@ -117,10 +117,25 @@ Using multiple `id segment`s might seem like unnecessary overhead at first. Addi
 - Using the key/value syntax (`id segment key` `:` `id segment value`) with [well-known `id segment keys`](well-known-id-segment-keys.md) enables `PAC-ID` routing in generic contexts.
 
 ## PAC-ID Extension
+`PAC-ID`s can be extended with custom information after a * character. The * character and subsequent characters are not part of the `PAC-ID` and can therefore freely be used for one or multiple `extension`s. In order to facilitate parsing and rendering by systems like [PAC-ID resolver](https://github.com/ApiniLabs/PAC-ID-Resolver), `name` and `type` of the `extension` MUST be specified.
```

Comment:
I'd regard name and type of extensions as optional because overall length is a concern. Name and type are mainly useful when there are multiple extensions present.